lib/service/Dialog.dart

```diff
@@ -168,126 +168,124 @@ Dialog wrongNetworkDialog(BuildContext context) {
 
 //dynamic
 Dialog walletDialog(BuildContext context) {
-  bool isWeb = true;
-  isWeb = kIsWeb && (MediaQuery.of(context).orientation == Orientation.landscape);
   Controller controller = Get.find();
   WalletController walletController = Get.find();
-  double _height = MediaQuery.of(context).size.height;
-  double _width = MediaQuery.of(context).size.width;
 
   return Dialog(
     backgroundColor: Colors.transparent,
     shape: RoundedRectangleBorder(
       borderRadius: BorderRadius.circular(12.0),
     ),
-    child: Container(
-      constraints: BoxConstraints(minHeight: 235, maxHeight: 250),
-      height: _height * 0.26,
-      width: _width * 0.20,
-      padding: EdgeInsets.all(20),
-      decoration: boxDecoration(Colors.grey[900]!, 30, 0, Colors.black),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.center,
-        children: <Widget>[
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              Text(
-                "Choose Wallet",
-                style: textStyle(Colors.white, 18, true),
-              ),
-              TextButton(
-                onPressed: () {
-                  Navigator.pop(context);
-                },
-                child: Icon(
-                  Icons.close,
-                  size: 30,
-                  color: Colors.white,
+    child: LayoutBuilder(
```

Comment:
Looks better. This should further be improved with a responsive one at some point.

Reply:
Thank you for the feedback. I learned something new today and I hope to continue learning more from you.

Reply:
We should all be learning something new every day.